content/surveys/_index.en.md

```diff
@@ -5,35 +5,94 @@ tags: ["wiki"]
+GPS based sampling in a known area).
+2. **Cluster based sampling**: most commonly used in combination with sampling villages proportional to population size.
+3. **Stratified samples**: uses either SRS or cluster based sampling but in two different adminstrative areas in order to obtain
+precise prevalence estimates for them.
+
+The templates can accomodate analysis of data from all sampling designs.
+
+## Sample size
+When calculating a sample size for your survey you take the below parameters into consideration:
+
+* **Estimate of prevalence**
+* **Design effect or intraclass coefficient**
+* **Precision required around your estimate of prevalence**
+
+You might consider recalculating these parameters once the survey is completed, in order to check that your original assumptions were accurate.
+
+### Estimate of prevalence
+You will need to use an estimate of the prevalence/coverage of the your main outcome of interest in your to-be-surveyed population (prevalence of malnutrition, measles vaccination coverage etc.). You will use published or grey literature to determine a logical estimate of this parameter.
+
+Note: the largest sample size you will need will be that which is calculated using an estimate of prevalence of 50%. The closer to 0% or 100% prevalence you set your estimate, the smaller your sample size will be. Therefore, if you have no idea what your estimate of prevalence is, try and use 50% and the largets possible sample size.
+
+### Design effect
+The easiest way to remember the design effect (which we usually refer to as DEFF), is that it is a measure of the variability of the outcome of interest within your clusters (if you are conducting a cluster based survey) and between your clusters. 
+
+The higher the DEFF, the higher the variability between clusters and the higher the probability that subjects within a cluster are similar.
+
+In a well designed survey, you will aim to have a calculated DEFF for your outcome of interest as close as possible to 1. This is because a DEFF of 1 means that your survey design has approached Simple Random Sampling (SRS) for your outcome of interest. SRS is the gold standard in survey design, but often not possible in MSF contexts. In general, in two-stage cluster sampling surveys, we would assume a DEFF of 2.
+
+Mathematically DEFF is calculated as:
+
+> **DEFF=1+(1-n) * rho**
+> where n=sample size of your survey and rho=intraclass coefficient
+
+### Precision around your estimate
+The precision around your estimate stipulates how close to the your estimate you would like the 95% confidence interval of your prevalence estimate to fall.
+
+> **For example**: you have calculated a 72% measles vaccination coverage in your study population. You would like to ensure that the 
+> 95% confidence interval is narrow (between 70-74%); thus you would set your precision at 2% (2% above and below 72%).
```

Comment:
I'm not sure how one would go about 'setting' precision. Is this something that the epi would declare in their report or before analyses?

Reply:
For the precsion it is only a question of the sample size.
So yes - this would be defined BEFORE the survey and used only to calculate the sample size.

Reply:
Can you add a commit re-wording the statement to make sure this is clear?